cloud/kubernetes/prometheus/prometheus.yaml

```diff
@@ -60,11 +60,12 @@ spec:
   - port: http
     path: /_status/vars
     tlsConfig:
-      # The HTTPS certs are signed by the kubernetes internal
-      # certificate authority.
-      caFile: "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
-      # This overrides the hostname verification check for the admin
-      # UI port to match our quickstart secure-mode cluster setup.
+      ca:
+        secret:
+          key: ca.crt
+          # This is the secret name used by the CockroachDB Kubernetes Operator.
+          # When using a custom CA, replace this with your secret name
+          name: cockroachdb-node
```

Comment:
@keith-mcclellan, is this something that we want to support in the long run or should we wait for the operator to have a stand alone ca.crt secret?

Reply:
This is an easy fix and will work indefinitely (we aren't removing the ca.crt from cockroachdb-node, just making a standalone secret that also hosts the file) so I'm okay with it.  We can't get that work done before the GA release of the operator anyway.